workbench/tools/pdf.py

```diff
@@ -615,3 +680,21 @@ def pdf_response(*args, **kwargs):
     activate(settings.WORKBENCH.PDF_LANGUAGE)
     kwargs["pdfdocument"] = PDFDocument
     return _pdf_response(*args, **kwargs)
+
+
+def get_debtor_address(postal_address):
+    address_lines = postal_address.splitlines()
+    country = "LI" if "LI" in address_lines[-1] else "CH"
+    if len(address_lines) >= 3:
+        return {
+            "name": " ".join(address_lines[0:-3])[:70],
+            "line1": address_lines[-2][:70],
+            "line2": address_lines[-1][:70],
+            "country": country,
+        }
+    else:
+        return {
+            "name": address_lines[0][:70] or "",
+            "line1": address_lines[1][:70] or "",
+            "country": country,
+        }
```

Comment:
Ah, das wars!

Ich frage mich, ob wir Rechnungsadressen auch in Deutschland etc. haben. Gewöhnlich können Rechnungen ins Ausland daran erkannt werden, dass sie nicht MwSt-pflichtig sind (`liable_to_vat`). Evtl. in diesen seltenen Fällen das Anhängen des QR Codes überspringen? Nur laut gedacht, vielleicht ist das egal.

Reply:
Hmm, ein ausländischer Kunde will vielleicht vom Schweizer Konto mit QR bezahlen.

Aber generell ist die Adresse ja nicht so ausschlaggebend bei der Generierung. Die Daten werden in der Bankapp ja ohnehin nur übernommen, mit dem Hinweis sie zu kontrollieren. Ich mach vielleicht einfach diese Helperfunktion stabiler.